docs/virtual_machines/net_binding_plugins/macvtap.md

```diff
@@ -0,0 +1,222 @@
+# Macvtap binding
+
+## Overview
+
+With the `macvtap` binding plugin, virtual machines are directly exposed to the Kubernetes
+nodes L2 network. This is achieved by 'extending' an existing network interface
+with a virtual device that has its own MAC address.
+
+Its main benefits are:
+
+- Direct connection to the node nic with no intermediate bridges.
+
+### Functionality support
+| Functionality                                  | Support |
+|------------------------------------------------|---------|
+| Migration support                              | No      |
+| IPAM support (on pod)                          | No      |
+| Require extra capabilities (on pod) to operate | No      |
+| Primary network (pod network)                  | No      |
+| Secondary network                              | Yes     |
+
+### Known Issues
+
+- Live migration is not fully supported, see [issue #5912](https://github.com/kubevirt/kubevirt/issues/5912#issuecomment-888938920)
+
+## Macvtap network binding plugin
+[v1.1.1]
+
+The binding plugin replaces the experimental core macvtap binding implementation
+(including its API).
+
+> **Note**: The network binding plugin infrastructure and the macvtap plugin
+> specifically are in Alpha stage. Please use them with care, preferably
+> on a non-production deployment.
+
+The macvtap binding plugin consists of the following components:
+
+- [Macvtap CNI plugin](https://github.com/kubevirt/macvtap-cni).
+
+The plugin needs to:
+
+- Deploy the CNI plugin binary on the nodes.
+- Deploy the Device Plugin daemon on the nodes.
+- Configure which node interfaces are exposed.
+- Define a NetworkAttachmentDefinition that points to the CNI plugin.
+- Enable the network binding plugin framework FG.
+- Register the binding plugin on the Kubevirt CR.
+- Reference the network binding by name from the VM spec interface.
+
+And in detail:
+
+### Macvtap CNI and DP deployment on nodes
+
+To simplify the procedure, use the
+[Cluster Network Addons Operator](https://github.com/kubevirt/cluster-network-addons-operator#macvtap)
+to deploy and configure the macvtap components in your cluster.
+
+The aforementioned operator effectively deploys the
+[macvtap](https://github.com/kubevirt/macvtap-cni) cni and device plugin.
+
+### Expose node interface to the macvtap device plugin
+
+There are two different alternatives to configure which host interfaces get
+exposed to the user, enabling them to create macvtap interfaces on top of:
+
+- select the host interfaces: indicates which host interfaces are exposed.
+- expose all interfaces: all interfaces of all hosts are exposed.
+
+Both options are configured via the `macvtap-deviceplugin-config` ConfigMap,
+and more information on how to configure it can be found in the
+[macvtap-cni](https://github.com/kubevirt/macvtap-cni#deployment) repo.
+
+This is a minimal example, in which the `eth0` interface of the Kubernetes
+nodes is exposed, via the `master` attribute.
```

Comment:
this is not up to date; The correct name of the attribute is `lowerDevice`.

https://github.com/kubevirt/macvtap-cni/pull/41

Reply:
Done